app/database/database.py

```diff
@@ -19,10 +20,28 @@ def init():
         logger.critical(e)
         exit(1)
 
-def get_member(gid, uid):
+
+def get_guild(gid):
```

Comment:
To keep things consistent, get_guild should probably also have a create_if_doesnt_exist=True parameter

Reply:
Refactored to `Guild.get_guild()`, like what Shaade did on the Member class